src/scripts/main.js

```diff
@@ -21,7 +21,20 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const rootList = document.createElement('ul');
+
+  for (const product in data) {
+    const item = document.createElement('li');
+
+    item.textContent = product;
+    rootList.appendChild(item);
+
+    if (Object.keys(data[product]).length) {
+      createTree(item, data[product]);
+    }
+  }
+
+  element.appendChild(rootList);
 }
```

Comment:
The 'createTree' function currently modifies the DOM within a loop which can cause performance issues if the data object is large. Consider creating a document fragment to build your list and then append it to the DOM once it is complete.